internal/trelloprovider/trello.go

```diff
@@ -0,0 +1,65 @@
+package trelloprovider
+
+import (
+	"context"
+	"fmt"
+	"github.com/adlio/trello"
+	"moul.io/multipmuri"
+	"moul.io/depviz/v3/internal/dvmodel"
+	"net/http"
+	"io/ioutil"
+	"strconv"
+	"github.com/tidwall/gjson"
+)
+
+func FetchCard(ctx context.Context, entity multipmuri.Entity, token string, apikey string, boardid string, out chan<- dvmodel.Batch) {
+	client := trello.NewClient(apikey, token)
+	board, err := client.GetBoard(boardid)
+	if err != nil {
+	  fmt.Println()
+	}
+	cards, _ := board.GetCards()
+	size := len(cards)
+	allcards := make([]string, size)
```

Comment:
```suggestion
	allcards := make([]string, len(cards))
```
may be simpler

Reply:
or not :)

Reply:
btw; what's the goal of this variable? it looks useless

Reply:
It was for testing, forgot to remove it. Thanks